mlflow/java/client/src/test/java/org/mlflow/tracking/ModelRegistryMlflowClientTest.java

```diff
@@ -13,18 +23,6 @@
 import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
 
-import java.io.File;
```

Comment:
intellij formatted this part of the file. Is there something like spotless that I should be running on these files to keep the import ordering correct?

Reply:
This is just fine with me :)